benches/lib.rs

```diff
@@ -127,7 +123,7 @@ fn bench_lib_consumer(b: &mut Bencher) {
         for _ in 0..10_000 {
             sender
                 .send(Some(WorkItem {
-                    format: ItemFormat::GCNO,
+                    format: ItemFormat::Gcno,
```

Comment:
This should be GCNO

Reply:
Actually not anymore after I fixed the clippy warnings back in the other PR.

As clippy recommends writing even acronyms in Pascal case/camel case. Like `HTML` becoming `Html` and `TCP` being `Tcp`.

We can if course roll that back but that would probably be better fit as a separate PR.